src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs

```diff
@@ -40,7 +40,7 @@ public async Task BitCompoundButtonShouldRespectIsEnabled(bool isEnabled, int co
                     parameters.Add(p => p.BitCompoundButtonStyle, style);
                 });
 
-            var bitButton = com.Find(".bit-compound-button");
+            var bitButton = com.Find(".bit-cmpnd-btn");
```

Comment:
"cmp" is a better abbriviation:  bit-cmp-btn

Reply:
bit-compound-button changed to bit-cmp-btn